inngest-spring-boot-adapter/src/main/java/com/inngest/springboot/InngestController.java

```diff
@@ -27,22 +28,16 @@ private HttpHeaders getHeaders() {
     @Value("${inngest.serveOrigin:}")
     private String serveOrigin;
 
-    @GetMapping()
+    @GetMapping
     public ResponseEntity<String> index(
         @RequestHeader(HttpHeaders.HOST) String hostHeader,
-        HttpServletRequest request
+        HttpServletRequest request,
+        @RequestHeader(name = "X-Inngest-Signature", required = false) String signature,
+        @RequestHeader(name = "X-Inngest-Server-Kind", required = false) String serverKind
     ) {
-        if (commHandler.getClient().getEnv() != InngestEnv.Dev) {
-            // TODO: Return an UnauthenticatedIntrospection instead when app diagnostics are implemented
-            return ResponseEntity.status(HttpStatus.FORBIDDEN)
-                .body("Introspect endpoint is only available in development mode");
-        }
-        String origin = String.format("%s://%s", request.getScheme(), hostHeader);
-        if (this.serveOrigin != null && !this.serveOrigin.isEmpty()) {
-            origin = this.serveOrigin;
-        }
-        String response = commHandler.introspect(origin);
-        return ResponseEntity.ok().headers(getHeaders()).body(response);
+        String requestBody = "";
```

Comment:
don't we need to get the actual request body somehow?

Reply:
hmm i believe it gave me an error trying to get the body of a GET request and later on trying it in cloud mode, it didn't seem to be sending a body. I'll try again to get it.

Reply:
Oh I forgot this is a GET, so yeah you're right there probably won't be a request body. Should we make the argument optional then?

Reply:
Also I made sure that the JS sdk returns an empty string for `GET` as well:
https://github.com/inngest/inngest-js/blob/56ed5c11081517db2a72ae27c83cbf4263d9b6ed/packages/inngest/src/components/InngestCommHandler.ts#L729-L735